spec/models/case/sar/offender_spec.rb

```diff
@@ -254,6 +280,81 @@
     end
   end
 
+  describe '#subject_name' do
+    it 'is the same as subject_full_name' do
+      kase = create :offender_sar_case
+      expect(kase.subject_name).to eq kase.subject_full_name
+      kase.update_attribute(:subject_full_name, "Bob Hope")
+      expect(kase.subject_name).to eq 'Bob Hope'
+    end
+  end
+
+  describe '#subject_address' do
+    it 'returns a dummy string for now' do
+      pending "Will's PR"
+      kase = create :offender_sar_case
+      expect(kase.subject_address).to eq 'Something sensible'
+    end
+  end
+
+  describe '#third_party_address' do
+    it 'wraps the postal_address field' do
+      kase = create :offender_sar_case
+      expect(kase.third_party_address).to eq kase.postal_address
+      kase.update_attribute(:postal_address, '11 The Road')
+      expect(kase.third_party_address).to eq '11 The Road'
+    end
+  end
+
+  describe '#requester_name' do
+    it 'returns third_party_name if third_party request' do
+      kase = create :offender_sar_case, :third_party
+      expect(kase.requester_name).to eq kase.third_party_name
```

Comment:
I'm not sure of the implementation here but, it is possible expectations on literal values might be better. As this is testing that the value of one method is eq to the value of another method on the same class. I think this can create issues in that what if both methods are "wrong" but are eq. Using literals means that we can be sure that `method_a` equals a known value without the proxy of the class.